presto-iceberg/src/test/java/com/facebook/presto/iceberg/TestIcebergMetadataListing.java

```diff
@@ -106,6 +106,15 @@ public void testTableListing()
         assertQuery("SHOW TABLES FROM iceberg.test_schema", "VALUES 'iceberg_table1', 'iceberg_table2', 'hive_table', 'iceberg_view'");
     }
 
+    @Test
+    public void testTableListingUsingInformationSchemaQuery()
```

Comment:
Have you verified that catalogs other than hive don't have this issue?

Reply:
IcebergNativeMetadata already has this fix. I also moved the test to IcebergDistributedTestBase, so it runs for all catalog type tests.